features/attached_enable.feature

```diff
@@ -114,6 +114,32 @@ Feature: Enable command behaviour when attached to an UA subscription
            | trusty  | libgit2-0 | https://esm.ubuntu.com/ubuntu/      |
            | xenial  | libkrad0  | https://esm.ubuntu.com/infra/ubuntu |
 
+    @series.xenial
+    @series.bionic
+    @series.focal
+    Scenario Outline: Attached enable of a know service shows update in a ubuntu machine
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token` with sudo
+        And I run `ua enable esm-infra` with sudo
+        Then I will see the following on stdout:
+            """
+            One moment, checking your subscription first
+            ESM Infra is already enabled.
+            See: sudo ua status
```

Comment:
do we need sudo in this status call? (not for this PR but sounds like a bug in our output/suggested help)

Reply:
That's a good point, I don't think we need that message. Do you remember with there is any context regarding that @blackboxsw ?

Reply:
We don't need sudo for `ua status` specifically, but when non-sudo runs of `ua status` rely on reading the cached status file @ /var/lib/ubuntu-advantage/status.json. So, if something specifically had changed state (like livepatch being disabled on the system, or rebooting into the proper hwe kernel for livepatch support on trusty) the status will lie to the non-root user. 
I think that was the reason we left 'sudo' on the status calls in messaging (there are 4 other sudo ua status messages in uaclient/status.py). 
It's not strictly required though... just that it current has caching caveats for infrequent corner cases

Reply:
Ah interesting ok. Maybe that's a bug then we can see if we can change or have the cache file permissions more free so that it's not necessary.

Reply:
@mitechie right. The issue that status has a heavily couple dependence on "machine-token.json" which contains authkeys that should only be presented to root. Also, any apt-service uses auth tokens stored in /etc/apt/auth.conf,d which is also only root-readable. So even checking entitlement_status as non-root would lie to a non-root user about whether the credentials to the PPAs are enabled because I think apt reports the ppa as not having the right credentials. So, we'd have to tweak how uaclient confirms enablement of apt-based entitlements to "work around" lacking root credentials for an apt-based service.